cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/ControlFlowSensitiveDFGPass.kt

```diff
@@ -498,6 +439,101 @@ open class ControlFlowSensitiveDFGPass(ctx: TranslationContext) : EOGStarterPass
         return state
     }
 
+    protected fun handlePartialAccessExpression(
```

Comment:
Maybe just a quick one-line comment here, with possibly a reference to the partial granularity thingie.

Reply:
We do not actually set the partial granularities here